### ISSUE TITLE
Issue with image tag of `ghcr.io/keptn-sandbox/keptn-service-template-go`

### ISSUE THREAD

**Issue by jainammm** [keptn-sandbox/keptn-service-template-go#158]
**Summary** 
Using Helm chart to `install keptn-service-template-go` give an error of pulling the image... It's because the `tag` is empty in file `values.yaml` of `chart` folder. There is no `latest` tag here [ghcr.io/keptn-sandbox/keptn-service-template-go](ghcr.io/keptn-sandbox/keptn-service-template-go). 

**Affected Component**

* [ ] Web UI / Bridge
* [ ] CLI
* [ ] REST API or webhooks
* [ ] User experience
* [ ] Developer experience
* [x] Deployment/manegement with Helm

**To Reproduce**

1. Run this command... `docker pull ghcr.io/keptn-sandbox/keptn-service-template-go`
2. No image will be pulled
3. `kubectly describe` and `kubectl logs` will also give more details when installing the whole service.

**Expected behavior**
To install the `keptn-service-template-go`

**How to Fix**
We can simply put this tag `0.17.1-dev` in `values.yaml` but that won't be sclable to other versions... We can also solve this by creating tag `latest` image on each release.

**Comment by christian-kreuzberger-dtx** [comment]
This is on purpose.
The Keptn service template itself is not meant to be installed, but the code serves as a template.

Regardless of that, you should just run `skaffold run` from within this repo, and that should build the container as well as set the correct image/tag setting for your helm chart.

Nevertheless, we will look into the helm chart and how we can improve it. It's not as simple as just setting "0.17.1-dev" as a tag in values.yaml, because we would have to update this tag every time we create a new release, and that's a lot of work, especially if you maintain more than one service.
Instead, we are providing CI, as well as pre-release and a release workflow, which are also building the helm chart with the correct tag.

In short,

* if you're developing locally, use `skaffold run`
* if you're consuming, download the helm chart from the releases page or from build artifacts of the CI workflow

**Comment by jainammm** [comment]
okay, thanks a lot for the clarification... yes sure its purpose is not to run but use it as a template... but the first thing i guess anyone would do is try to run this template and see how it's behaving and then dive deeper... so i thought there should be a `latest` tag build

closing this issue for now then
